packages/css/src/atoms/textarea/_utils.scss

```diff
@@ -0,0 +1,95 @@
+@use "../../utility/index" as utils;
+@use "../../themes/%textarea%/constants/colors.scss" as theme_colors;
+
+@function getIdleColor($colors){
+    @return map-get($colors, "idle" );
+}
+
+@function getErrorColor($colors){
+  @return map-get($colors, "error" );
+}
+
+@function getSuccessColor($colors){
+  @return map-get($colors, "success" );
+}
+
+@function getActiveColor($colors){
+  @return map-get($colors, "active")
+}
```

Comment:
Please fix indentations, missing semicolons and spacing to keep everything consistent.

Reply:
added!